conversation/ui
# TC-414 Add Dialog docs

This PR adds documentation for the `<Dialog>` component.

## How to test

* Run `make storybook`
* Open `localhost:9001` and check it out

![screenshot_2018-12-10 storybook 1](https://user-images.githubusercontent.com/3614/49705640-f19f7d00-fc73-11e8-8928-226f62616e01.png)
